packages/styles/scss/patterns/blocks/content-block-segmented/_content-block-segmented.scss

```diff
@@ -0,0 +1,59 @@
+//
+// Copyright IBM Corp. 2016, 2018
+//
+// This source code is licensed under the Apache-2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+//
+@import '../../../globals/imports';
+
+@mixin content-block-segmented {
+  %block-image {
+    width: 100%;
+    max-width: rem(640px);
+  }
+
+  .#{$prefix}--content-block {
+    &-segmented {
+      .#{$prefix}--content-group {
+        &__children {
+          .#{$prefix}--image__img {
```

Comment:
Is this much specificity necessary?

Reply:
It definitely shouldn't be. There are a lot of styles in the wrong place that we need to fix.